src/scripts/popup.js

```diff
@@ -275,14 +275,18 @@ window.PopUp = {
     }
 
     if (startTimer) {
-      if (!PopUp.$timer) {
-        PopUp.$timer = setInterval(function () {
-          PopUp.showCurrentDuration();
-        }, 1000);
-      }
+      clearInterval(PopUp.$timer);
```

Comment:
Semantically I'd prefer a call to the stop method
```suggestion
     PopUp.stopDurationInput();
```